robyn/__init__.py

```diff
@@ -51,6 +52,11 @@ def __init__(
         self.dependencies = dependencies
         self.openapi = openapi
 
+        if openapi_file_path:
+            openapi.set_json_spec(self.directory_path + "/" + openapi_file_path)
+        elif os.path.isfile(self.directory_path + "/openapi.json"):
+            openapi.set_json_spec(self.directory_path + "/openapi.json")
+
```

Comment:
@VishnuSanal , could you explain the logic here? What is this supposed to do? 

It should just be a file.

Reply:
this should invoke the override function if the path param is set or if the file i9s present in the pre-defnked path